student.rb

```diff
@@ -1,9 +1,12 @@
 require './person'
 
 class Student < Person
+  attr_reader :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.students << self
```

Comment:
- You have not properly implemented the `belongs-to` relationship between the student class and the classroom class. Please refer to this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more info.